SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKSimulatedPlayerDeployment.cpp

```diff
@@ -506,6 +507,11 @@ void SSpatialGDKSimulatedPlayerDeployment::OnDeploymentAssemblyCommited(const FT
 	SpatialGDKSettings->SetAssemblyName(InText.ToString());
 }
 
+void SSpatialGDKSimulatedPlayerDeployment::OnProjectNameCommitted(const FText& InText, ETextCommit::Type InCommitType)
+{
+	FSpatialGDKServicesModule::SetProjectName(InText.ToString());
```

Comment:
Can we do a regex pass on this please - Users will trip up guaranteed

Reply:
Huh, so apparently we have a `IsProjectNameValid` function for that but it's not currently used. It looks like everything else only gets regex checked when you click the launch button but I guess this is different because we write the result into `spatialos.json` instead of just changing the variable.